examples/ota-provider-app/ota-provider-common/BdxOtaSender.cpp

```diff
@@ -82,13 +82,18 @@ void BdxOtaSender::HandleTransferSessionOutput(TransferSession::OutputEvent & ev
             // end of the transfer.
             sendFlags.Set(chip::Messaging::SendMessageFlags::kExpectResponse);
         }
-        VerifyOrReturn(mExchangeCtx != nullptr, ChipLogError(BDX, "%s: mExchangeCtx is null", __FUNCTION__));
+        VerifyOrReturn(mExchangeCtx != nullptr, ChipLogError(BDX, "mExchangeCtx is null"));
```

Comment:
This should never happen... Wonder why we log here. It applies to every single part of every protocol

Reply:
Removed the log